README.md

```diff
@@ -12,27 +12,19 @@ This is still a work-in-progress
 
 In CoreDHCP almost everything is implemented as a plugin. The order of plugins in the configuration matters: every request is evaluated calling each plugin in order, until one breaks the evaluation and responds to, or drops, the request.
 
-The following configuration runs a DHCPv6-only server, listening on all the interfaces, using a custom DUID-LL as server ID, and reading the leases from a text file.
+The following configuration runs a DHCPv6-only server, listening on all the interfaces, using a custom server ID and DNS, and reading the leases from a text file.
 
 ```
 server6:
     listen: '[::]:547'
```

Comment:
if we're going for minimum viable configuration, we can even just remove this line
This is even misleading because specifying the address manually will result in not actually joining the default multicast groups (whereas not having the `listen` at all will result in joining all_dhcp_servers and all_dhcp_servers_and_relays; but not listening on the unicast addresses)

I'll send a PR with better doc in the example config for the listen statement, I realize it's underdocumented

Reply:
How do we join the default multicast groups with an explicit line? I think we should have that

Reply:
Btw right now I'll remove the line, and add a comment that no `listen` directive will listen on all interfaces on the default multicast group (and we can expand on other syntaxes in the example config)

Reply:
You just write them out:
```yaml
listen:
  - "[ff02::1:2]" # This will automatically expand to joining the group on each existing interface
  - "[ff05::1:3]"
```

Reply:
Updated again

Reply:
> You just write them out

Cool :) this should go to `config.yml.example`